.gitignore

```diff
@@ -102,3 +102,26 @@ dist
 
 # TernJS port file
 .tern-port
+
+
+
+# Created by https://www.toptal.com/developers/gitignore/api/react
+# Edit at https://www.toptal.com/developers/gitignore?templates=react
+
+### react ###
+.DS_*
```

Comment:
Not sure what this line does, can you explain it?

Reply:
This is for some files on Mac OSX